docs/user-guide/profile.md

```diff
@@ -119,24 +128,70 @@ When you're in Profle page, you can see your identity list
 ![image](https://github.com/linagora/tmail-flutter/assets/68209176/5e50a51d-024f-4bb1-abd9-2dd10dbb0ca0)
 
 
-## Vacation
+## Vacation mode
 
 - Automatic vacation replies are messages that are sent automatically to anyone who emails you while you're on vacation.
 - Setting up automatic vacation reply is a convenient way to inform your contacts that you're away and won't be able to respond to emails promptly.
-- On Manage accout page, YOu can select tab "Vacation" on left meny to access vacation reply settings
+- On Manage accout page, YOu can select tab "Vacation" on left menu to access vacation reply settings
+
+![image](https://github.com/linagora/tmail-flutter/assets/68209176/f2743333-bc42-4cc7-a50e-81121c43a355)
+  
+### 1. Vacation setting 
 
```

Comment:
This serction enable you to enable an automatic responder, that can be used to answer people automatically, tell them when to come back, and how to manage their issues when you are not here.